src/psij/executors/batch/batch_scheduler_executor.py

```diff
@@ -509,30 +513,34 @@ def _poll(self) -> None:
         logger.info('Polling for %s jobs', len(jobs_copy))
         try:
             out = self.executor._run_command(self.executor.get_status_command(jobs_copy.keys()))
-            logger.debug('Output from status command: %s', out)
+        except subprocess.CalledProcessError as ex:
+            out = ex.output
+            exit_code = ex.returncode
+        else:
+            exit_code = 0
             self._poll_error_count = 0
+        logger.debug('Output from status command: %s', out)
+        try:
+            status_map = self.executor.parse_status_output(exit_code, out)
```

Comment:
This seems to be also called when the above `try` raised an exception - is that the intent?